ci-operator/README.md

```diff
@@ -0,0 +1,153 @@
+ci-operator
+===========
+
+
+End-to-end tests
+----------------
+
+The preferred way to write e2e tests is using `ci-operator`.  See the
```

Comment:
We should define what `e2e` means

Reply:
"Tests that require deploying a temporary cluster"?

Reply:
Moreso tests of k8s internal functionality from the end-user level or of applications on top of a k8s cluster